tools/test-proxy/Azure.Sdk.Tools.TestProxy/Azure.Sdk.Tools.TestProxy.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
```

Comment:
Is .NET 5 actually interesting any longer that you moved to 6?

Reply:
Is .NET 5 available on the agents by default now? If so, I'm fine with dropping support on .NET 5.

Reply:
Yes .NET 6.0 is available currently on all the agents.

Reply:
Excellent. Dropping .NET 5 then!